### PR TITLE
tbdev turndown: replace dev subcommand with shim

### PR DESCRIPTION
**Pull request by nfelt**
This replaces the `dev` subcommand with a shim that prints out a message about the turndown.

Unfortunately, the actual uploader code can't quite be deleted yet because Cloud AI Platform is depending on it still:
https://github.com/googleapis/python-aiplatform/blob/cd31c1306a9a00a01fbc1dda56fe99ed567a4cfb/google/cloud/aiplatform/tensorboard/uploader.py#L62-L65

### CODE REVIEW

File: tensorboard/main.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def run_main():
 
     tensorboard = program.TensorBoard(
         plugins=default.get_plugins(),
-        subcommands=[uploader_subcommand.UploaderSubcommand()],
+        subcommands=[legacy_tbdev_subcommand.LegacyTbdevSubcommand()],
```
Comment (bmd3k) [inline tensorboard/main.py:43]: Maybe it's not worthwhile but it seems like you could at least delete uploader_subcommand.py.

**Review by bmd3k — APPROVED** [review]

**Comment by bmd3k** [comment]
Also, out of curiosity, why develop this in the tensorflow/tensorboard repo instead of your own repo?

File: tensorboard/legacy_tbdev_subcommand.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+class LegacyTbdevSubcommand(program.TensorBoardSubcommand):
+    """Legacy `tensorboard dev` implementation to print shutdown info."""
+
+    def name(self):
+        return "dev"
+
+    def define_flags(self, parser):
+        pass  # no flags to define
+
+    def run(self, flags):
```
Comment (bmd3k) [inline tensorboard/legacy_tbdev_subcommand.py:41]: This works great for `tensorboard dev` but for `tensorboard dev upload` or `tensorboard dev auth revoke` you get a somewhat-less-helpful error message like "tensorboard: error: unrecognized arguments: upload".

Is that intentional?

Very few people will just run `tensorboard dev`.

**Review by bmd3k — COMMENTED** [review]
Sorry for

**Review by nfelt — COMMENTED** [review]
> Also, out of curiosity, why develop this in the tensorflow/tensorboard repo instead of your own repo?

I usually prefer my own repo for one-off PRs, but you can only make a given PR a "diffbase" of another PR if its branch is in the main repo, since that allows it to be the merge target for the subsequent PR and generate the diffs relative to it during review (and then to submit you would re-point it to merge into master).

So I did that in this case under the assumption that I would stage the rest of the deletion as its own child PR, but then didn't quite get around to that.

**Review by bmd3k — APPROVED** [review]

**Pull request merged**